frequency               : 10  # [Hz]

service_recorder_start  : wpRecorder/start
service_recorder_end    : wpRecorder/end

lookupTimeout           : 0.2 # [s]

frame_id_robot          : base_link
frame_id_map            : map

file_dir                : /home

pitch                   : 1.0 # [m]